src/components/Home/HothDescription.jsx

```diff
@@ -52,8 +52,7 @@ export default function HothDescription() {
 						During HOTH, you&apos;ll get to learn from{' '}
 						<strong>workshops</strong>, receive technical help from{' '}
 						<strong>mentors</strong>, and meet new people while participating in{' '}
-						<strong>fun</strong>
-						social activities. There will be <strong>prizes</strong> for the
+						<strong>fun</strong> social activities. There will be <strong>prizes</strong> for the
```

Comment:
I don't think we need to reformat this part, let's keep it as it normally was. My bad, there should be a vertical ruler defined somewhere in our linting rules... I'll look more closely at that later, but if it doesn't exist I'll create an issue for it.